src/server/index.jsx

```diff
@@ -104,14 +104,20 @@ server
 
       const headHelmet = Helmet.renderStatic();
 
-      const styleTags = sheet.getStyleElement();
+      let inlineCss = '';
+      try {
+        // eslint-disable-next-line prefer-destructuring
+        inlineCss = renderToString(sheet.getStyleElement()).match(
```

Comment:
If `sheet.getStyleElement()` returns an array with 2 elements inside, instead of 1 (which it theoretically can), This regex will break and only catch the opening tag for the 1st one, and closing tag for the second, meaning invalid css